proxy.js

```diff
@@ -24,12 +24,28 @@
   let lastRevokeFn = null;
 
   /**
+   * @param {*} o
+   * @return {boolean} whether this is probably an Object
+   */
+  function isObject(o) {
+    return o instanceof Object || (!!o && (typeof o == 'object' || typeof o == 'function'));
+  }
+
+  /**
+   * @param {*} o
+   * @return {boolean} whether this is probably a Function
+   */
+  function isFunction(o) {
```

Comment:
since this is used only once, I will recommend to remove this method, and just do a typeof inline